docs/application/vscode-ext/web.md

```diff
@@ -0,0 +1,306 @@
+  2. Click **Install** to select profiles and versions that you need for your project.
+
+     ![Package Manager](media/tizen_package_manager.png)
+
+- To use Tizen Emulator Manager, follow these steps:
+
+  > **Note**  
+  >
+  > Tizen Emulator Manager shows the emulator images installation window only when no images are installed on your computer.
+
+  1. Open **Command Palette** and enter **Run Emulator-Manager**.
+  2. Select the profiles and versions that you want to install and click **Ok**.
+
+     ![Emulator Manager](media/howtoinstall-emulatormanager.png)
+
+
+## Develop Applications
+
+The following section explains how to use VS Code Extension for Tizen Web to develop the applications:
+
+### Create Tizen Web Project
+
+To create a Tizen Web project, follow these steps:
+
+1. Open **Command Palette** and enter **Create Wizard Project**.
+
+   ![Create Wizard Project](media/tizen_web7.png)
+
+2. Select the **Template** project type and click **Next**.
+
+   ![Select the type of project](media/tizen_web8.png)
+
+3. Select the application profile and version from the drop-down list and click Next.
+
+   ![Select the profile and version](media/tizen_web9.png)
+
+4. Select the **Basic UI** template and click **Next**.
+
+  > **Note**
+  >
+  > The list is different depending on the type and profile of the previously selected project.
+
+  ![Select the project](media/tizen_web10.png)
+
+5. Enter the **Project name** and click **Finish** to complete the project creation in workspace.
+  > **Note**
+  >
+  > packageId is an optional field, it is not mandatory to enter details in this field because the packageId will generate the details automatically.
+
+  ![Enter the name of project](media/tizen_web11.png)
+
+6. If you checked all this, the project is created.
+
+   ![Project is created](media/tizen_web12.png)
+
+### Creating Tizen Certificate
+
+To create Tizen Certificate, follow these steps:
+
+  1. Open **Command Palette** and enter **Create Tizen Certificate**.
+  2. Enter the name of the certificate.
+  3. Enter the password of the certificate.
+  4. Enter the profile name of the certificate. (Extension is .p12)
+  > **Note**
+  > 
+  > If it is created properly, profilename.p12 will be created in tizen-studio/keystore/author.
+
+To add Tizen Certificate, follow these steps:
+
+  1. Open **Command Palette** and enter **Add Tizen Certificate**.
+  2. Enter the name of the certificate.
+  3. Enter the password of the certificate.
+  4. Enter the profile name of the certificate. (Extension is .p12)
+  > **Note**
+  > 
+  > You can verify the added certificate by running **Certificate Manager**.
+
+To Remove your Tizen certificate:
+
+If you want to change the Certificate, you need to remove the added Certificate.
+
+  1. Open **Command Palette** and enter **Remove Tizen Certificate**.
+  2. Enter the name of the certificate.
+  > **Note**
+  >
+  > When you run Certificate Manager the certificate will be deleted.
```

Comment:
@annie-samsung Would you please also see this with @sabeena21?

Reply:
@sabeena21 : The meaning is altered in your rewording, please ensure not do so.
@mijin85cho,
In my opinion, this Note is not necessary because it is a negative action, to go and see that certificate is not listed anymore.

**OPTION 1**>> However, if this Note has to be retained, here is my suggestion:

Enter **Run Certificate-Manager** on the **Command Palette**, you can notice that the certificate is not listed on the Tizen Certificate Manager window.

____________________________________________________________________________________
**OPTION 2**>> Instead of Note,  I would recommend adding a line at the end of the two steps like a result or conclusion, such as:

If you want to change the Certificate, you need to remove the added Certificate.

  1. Open **Command Palette** and enter **Remove Tizen Certificate**.
  2. Enter the name of the certificate.

The certificate is now removed from the Tizen Certificate Manager window. To view the Tizen Certificate Manager window, enter **Run Certificate-Manager** on the **Command Palette**.
 
____________________________________________________________________________________
**OPTION 3**>> should we say "specific certificate" instead of just "certificate" in OPTION 1/2

you can notice that the specific certificate is not listed
or
The specific certificate is now removed

Reply:
@annie-samsung Thanks for the checking. I agree to your comment that we don't need the note.
Among your suggestions, I think the second option, to make the sentence as the part of the paragraph is better than others.

The specific certificate is now removed from the Tizen Certificate Manager window. To view the Tizen Certificate Manager window, enter Run Certificate-Manager in the Command Palette.

Question to @annie-samsung. Did you personally check the command **Run Certificate-Manage**?